### PR TITLE
Created form to create a new game and made migration to include game nam...

### PR DESCRIPTION
**Pull request by Neilda**
Created a form to create a new game and made a migration to include a game name.

### CODE REVIEW

File: app/models/game.rb [hunk 1]
```diff
@@ -1,4 +1,6 @@
 class Game < ActiveRecord::Base
 	has_many :users
 	has_many :pieces
+
+	devise :database_authenticatable, :registerable, :recoverable, :rememberable, :trackable, :validatable
```
Comment (kenmazaika) [inline app/models/game.rb]: I don't think this is quite right.  A "game" can't login via devise.

File: test/controllers/games_controller_test.rb [hunk 2]
```diff
@@ -1,7 +1,22 @@
...
+		assert_redirected_to new_user_session_path
+	end
+
+	test "someone is logged in- new" do
+		user = FactoryGirl.create(:user)
+		sign_in user
+		get :new
+		assert_response :success
+	end
+
```
Comment (kenmazaika) [inline test/controllers/games_controller_test.rb:19]: @Neilda, can you add a test for index as well?

**Comment by kenmazaika** [comment]
It looks like the pull request can't be merged with the button, because a merge conflict would happen when merging in master.  Can you pull the latest master code into your branch?

**Comment by kenmazaika** [comment]
I made some notes, but awesome work, @Neilda.  Keep is up! :)

**Comment by Neilda** [comment]
Thanks for the notes, Ken. I’ll rework on it again.

—

**Pull request merged**